CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.12)
-project(ArborX CXX)
+project(ArborX CXX CUDA)
```

Comment:
Note: this is needed for cmake to pick up `.cu` files. Otherwise, they are completely ignored.

Reply:
But you only want to do this if we actually build with `CUDA` support (see the `clang` and `PGI` builds), don't you?

Reply:
There is no downside to always having this on, is there?

Reply:
I am suspecting that these two builds fail because of it, see https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/ArborX/detail/PR-237/3/pipeline/15 and https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/ArborX/detail/PR-237/3/pipeline/16.

Reply:
> I guess I need to switch it to `enable_language(CUDA)` hidden behind `Kokkos_ENABLE_CUDA`.

Yes, I was just about to suggest that.